test/cypress/integration/group-proposals.spec.js

```diff
@@ -271,23 +282,23 @@ describe('Proposals - Add members', () => {
       .should('contain', 'Portuguese classes')
   })
 
-  it('It should remove non monetary contribution', () => {
+  it('User 1 remove non monetary contribution', () => {
```

Comment:
typo, missing the s, removes*. Same goes for other test descriptions, edits, removes, gives, etc..